packages/babel-plugin-transform-regenerator/test/fixtures/misc/arrow-inside-generator/output.js

```diff
@@ -0,0 +1,17 @@
+var _marked = /*#__PURE__*/regeneratorRuntime.mark(gen);
```

Comment:
This isn't a new test, but the tests folder had an invalid layout so this file was never generated.